dbsync-receiver-app/pom.xml

```diff
@@ -33,7 +33,11 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
-                    <attach>false</attach>
+                    <!-- will generate 2 jars:  -->
+                    <!-- the normal jar usable as a dependency-->
+                    <!-- an executable jar with the classifier exe-->
+                    <attach>true</attach>     
+                    <classifier>exe</classifier>
                 </configuration>
```

Comment:
@wluyima  please see the update

Reply:
attach=true is the default, so now we can just remove it since we are using the classifier

Reply:
We also need to update the dbsync readme files in the installation instructions to say which runnable jar file to use